dashboard/dashboard.py

```diff
@@ -34,19 +109,27 @@ def report_download_page() -> None:
 def subscribe_page() -> None:
     """
     Creates subscribe page where users can subscribe
-    to daily summary PDF reports and/or alerts about
+    to daily summary PDF reports and / or alerts about
     trends and genres via email.
     """
     st.title("Subscribe Page")
-    st.write("Subscribe to updates and alerts below.")
+    st.write("Subscribe to daily reports and (optionally) alerts below.")
 
     # Email input
     email = st.text_input("Enter your email address:")
 
-    # Checkboxes for user subscriptions
-    subscribe_daily = st.checkbox("Subscribe to daily reports")
+    # Checkbox for user alerts
     subscribe_alerts = st.checkbox("Subscribe to alerts")
```

Comment:
If we have time to spare we could think about adding a checkbox daily report subscriptions so that users can subscribe to alerts without also having to subscribe to email reports.

Reply:
I agree, will see what time is looking like early next week.